crates/types/src/services/executor.rs

```diff
@@ -131,6 +131,13 @@ impl<DatabaseTransaction> UncommittedResult<DatabaseTransaction> {
 pub enum Event {
     /// Imported a new spendable message from the relayer.
     MessageImported(Message),
+    /// Ignored a message from the relayer.
+    MessageIgnored {
+        /// Ignored message
+        message: Message,
+        /// The reason for ignoring the message
+        reason: String,
+    },
```

Comment:
We shouldn't add a new event at the middle since it is a breaking change for the WASM interface, basically you break all nodes in the network with this update and forces them to use a new `fuel-core` node.

Plus, I don't think that we even need `MessageIgnored` event since it doesn't change the state of the chain.

Reply:
That's fair. It was an afterthought and added a breaking section the the CHANGELOG to flag for this reason.